src/Analyzers/MSTest.Analyzers/xlf/Resources.ja.xlf

```diff
@@ -520,6 +520,21 @@
         <target state="translated">テスト メソッドを無視することはできません</target>
         <note />
       </trans-unit>
+      <trans-unit id="TypeContaningTestMethodShouldBeATestClassDescription">
```

Comment:
Conta **i** ning. 

This typo is in many places. Both in text and in type name.